tests/infra/node.py

```diff
@@ -397,7 +417,9 @@ def signing_auth(self, name=None):
             "signing_auth": self.identity(name),
         }
 
-    def client(self, identity=None, signing_identity=None, **kwargs):
+    def client(
+        self, identity=None, signing_identity=None, interface_idx=None, **kwargs
```

Comment:
I think we almost certainly want to name interface.

Reply:
I'm not sure what you mean here, you want the arg to be `interface` (and eventually an identifiable string) rather than `interface_idx`?

Reply:
Yes, I think this will be useful for definition, monitoring etc. But we can re-visit that once we change the config format.